db/migrate/20220816184148_create_loans.rb

```diff
@@ -0,0 +1,15 @@
+class CreateLoans < ActiveRecord::Migration[7.0]
+  def change
+    create_table :loans do |t|
+      t.string :loan_type
+      t.string :creditor
+      t.string :description
+      t.string :user
```

Comment:
This should be a foreing key `user_id` pointing to the `users` table

Reply:
@ernest-phillips you can check the [`:references`](https://guides.rubyonrails.org/getting_started.html#adding-a-second-model-generating-a-model) keyword to tackle this

Reply:
Both `creditor` and `user` should be a reference to their respective tables

I see you have migrations for `loans` and `users`, but no migration to create `creditors`, before you add `t.references :creditor, null: false, foreign_key: true` you should create the `Creditor` model and it's table